scripts/TRENDY_runs/v12/run_TRENDY.sh

```diff
@@ -24,7 +24,7 @@ module load geos/3.8.0
 module load intel-compiler/2021.8.0
 export R_LIBS=/g/data/x45/R/libs
 export PYTHONPATH=/g/data/x45/python3.10.4/lib/python3.10/site-packages
-
+# module load conda_concept/analysis3       # Can load python libraries from this module, supplied by project hh5
```

Comment:
Can you add in the comment that access to hh5 can be asked on [https://my.nci.org.au/](https://my.nci.org.au/)?